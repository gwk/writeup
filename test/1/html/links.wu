writeup v0

<http://github.com> <http://github.com github>
<https://github.com> <https://github.com github>
<link:test/assets/text.txt> <link:test/assets/text.txt text>
<mailto:a@b.c> <mailto:a@b.c address>
