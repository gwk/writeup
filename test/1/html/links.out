<p>
  <a href="http://github.com">http://github.com</a> <a href="http://github.com">github</a>
<br />
  <a href="https://github.com">https://github.com</a> <a href="https://github.com">github</a>
<br />
  <a href="test/assets/text.txt">test/assets/text.txt</a> <a href="test/assets/text.txt">text</a>
<br />
  <a href="mailto:a@b.c">mailto:a@b.c</a> <a href="mailto:a@b.c">address</a>
</p>
